Doc/library/ftplib.rst

```diff
@@ -55,17 +55,44 @@ Reference
 FTP objects
 ^^^^^^^^^^^
 
-.. class:: FTP(host='', user='', passwd='', acct='', timeout=None, source_address=None, *, encoding='utf-8')
-
-   Return a new instance of the :class:`FTP` class.  When *host* is given, the
-   method call ``connect(host)`` is made.  When *user* is given, additionally
-   the method call ``login(user, passwd, acct)`` is made (where *passwd* and
-   *acct* default to the empty string when not given).  The optional *timeout*
-   parameter specifies a timeout in seconds for blocking operations like the
-   connection attempt (if is not specified, the global default timeout setting
-   will be used). *source_address* is a 2-tuple ``(host, port)`` for the socket
-   to bind to as its source address before connecting. The *encoding* parameter
-   specifies the encoding for directories and filenames.
+.. class:: FTP(host='', user='', passwd='', acct='', timeout=None, \
+               source_address=None, *, encoding='utf-8')
+
+   Return a new instance of the :class:`FTP` class.
+   When *host* is given, the method call :meth:`connect(host) <connect>`
+   is made by the constructor.
+   When *user* is given, additionally the method call
+   :meth:`login(user, passwd, acct) <connect>` is made
+   (where *passwd* and *acct* default to the empty string when not given).
+
+   :param str host:
+      The hostname to connect to.
```

Comment:
Perhaps move the info about _host_ from the opening paragraph to here? I can see a reason not to, so this section isn't too "overloaded".

Reply:
I considered that, but it felt weird after trying it out; the two sentences that describe the implicit constructor behaviour are tightly connected.